src/test/groovy/nebula/plugin/BaseIntegrationTestKitSpec.groovy

```diff
@@ -4,6 +4,16 @@ import nebula.test.IntegrationTestKitSpec
 
 abstract class BaseIntegrationTestKitSpec extends IntegrationTestKitSpec {
     def setup() {
+        // Enable configuration cache :)
```

Comment:
@OdysseusLives : this enables config cache for all tests and added a helper method to disable in some cases